spark/src/test/java/org/apache/iceberg/spark/data/AvroDataTest.java

```diff
@@ -185,4 +188,18 @@ public void testMixedTypes() throws IOException {
 
     writeAndValidate(schema);
   }
+
+  @Test
+  public void testTimestampWithoutZone() throws IOException {
+    Option<SparkSession> sparkSession = SparkSession.getActiveSession();
+    if (sparkSession.isDefined()) {
+      sparkSession.get().conf().set(SparkUtil.HANDLE_TIMESTAMP_WITHOUT_TIMEZONE, "true");
+    }
```

Comment:
This is in `AvroDataTest`, but there is some code for handling additional Spark SQL conf options here that might be something worth copying:

https://github.com/apache/iceberg/blob/25eaebacbd1e250d2d884b49fc753a23b6aa6eaf/spark/src/test/java/org/apache/iceberg/spark/SparkTestBase.java#L171-L206

Additionally, it might be good to unset this property after its use (so the configuration doesn't get mixed up with other tests now or in the future).

Reply:
good point, I changed the code based on your recommendations